bentoml/_internal/configuration/__init__.py

```diff
@@ -99,6 +95,18 @@ def get_debug_mode() -> bool:
     return False
 
 
+def set_quiet_mode(enabled: bool) -> None:
+    os.environ[DEBUG_ENV_VAR] = str(enabled)
+
+    # do not log setting quiet mode
```

Comment:
should it also call `configure_logging` here?  If the user calls this after a `configure_logging`, the env var won't be applied to the logging module?

Reply:
I removed the `configure_logging` calls from both of these since when users use this as a library `configure_logging` should never be called.

Reply:
We could have a flag that checks whether `configure_logging` has been called and call this again if the user has called that, but I think this is ok; `configure_logging` is an internal API right now anyway.

Reply:
Sounds good, if users need to customize the logging, they can probably do it via the standard logging library as well.